### ISSUE TITLE
Screen Reader does not report direction when items are flipped.

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#30876]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

In version 4.6 a number of keys now announce their function when pressed.  Pressing X to flip an item now results in speech feedback that says "flipped direction".  But this does not tell the user whether the item was flipped up or down, e.g. whether the stem is now up or down or the item is now above or below the staff.  There is also no other way to find this information out.

Ideally key presses should always announce the action that has taken place.  In this instance "flipped Up", or "flipped down".

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.3

### Regression

I was unable to check

### Operating system

windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
